diff --git a/install_ascend.sh b/install_ascend.sh
index 48785a9..ed9187c 100644
--- a/install_ascend.sh
+++ b/install_ascend.sh
@@ -162,7 +162,7 @@ MAKE_JOBS=$(( CORES * 50 / 100 ))
 if [ $MAKE_JOBS -lt 1 ]; then
     MAKE_JOBS=1
 fi
-make -j${MAKE_JOBS}
+make -j
 
 if [ $? -ne 0 ]; then
     echo "Error: TileLang build failed."
diff --git a/requirements-build.txt b/requirements-build.txt
index 784cb60..4c2318f 100644
--- a/requirements-build.txt
+++ b/requirements-build.txt
@@ -1,6 +1,5 @@
 # Should be mirrored in pyproject.toml
 build
-cmake>=3.26
 packaging
 setuptools>=61
 torch
